### PR TITLE
Create data element if mapping key not in dicom

### PR DESCRIPTION
**Pull request by NPann**

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -400,6 +400,16 @@ def _modify_dicom_archive(dicom_file_path, update_keys, flywheel_dicom_header, d
...
                     log.warning('{} key is empty in Flywheel [{}={}]. DICOM header will remain [{}={}]'.format(key, key, flywheel_dicom_header.get(key), key, dicom.get(key)))
+            else:
+                if pydicom.datadict.tag_for_keyword(key):  # checking keyword is valid
+                    log.info(
+                        '{} data element not present in DICOM header. Creating it.'.format(
+                            key))
+                    setattr(dicom, key, flywheel_dicom_header.get(key))
+                else:
+                    log.warning(
+                        'Unknown DICOM keyword: {}. Date element will not be created.'.format(
```
Comment (lmperry) [inline run.py:411]:
```suggestion
                        'Unknown DICOM keyword: {}. Data element will not be created.'.format(
```

**Review by lmperry — APPROVED** [review]
LGTM. One typo to fix.

**Review by kalebdfischer — APPROVED** [review]
LGTM.

**Comment by NPann** [comment]
Test added. Merging.

**Pull request merged**